user_tools/src/spark_rapids_pytools/resources/qualification-conf.yaml

```diff
@@ -233,8 +233,8 @@ local:
         - 'Medium'
         - 'Large'
       sortingColumns:
-        - 'Estimated GPU Speedup'
-        - 'Unsupported Operators Stage Duration Percent'
+        - 'Speedup Category Order'
+        - 'Total Core Seconds'
```

Comment:
Should we sort by 3 columns? We still use the unsupported percent column.

Reply:
Changed sorting columns to 'Speedup Category Order', 'Total Core Seconds', and 'Unsupported Operators Stage Duration Percent', but I think it is unlikely that we need to break ties after 'Total Core Seconds'.